src/runtime/d3d12compute.cpp

```diff
@@ -194,6 +197,10 @@ static T zero_struct() {
     return zero;
 }
 
+#define hashmap_malloc  d3d12_malloc
+#define hashmap_free    d3d12_free
+#include "hashmap.h"
+
```

Comment:
Ah, now I understand why there's a preprocessor definition in `hashmap.h`.  Please add an explanatory comment there so nobody accidentally changes it.

Reply:
👍